File: src/main/java/com/rs/java/game/area/multi/GodwarsMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class GodwarsMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Godwars Multi";
	}

	public GodwarsMulti() {
		super(11345, 11346, 11347, 11601, 11602, 11603);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/GodwarsMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class GodwarsMulti : Area(11345, 11346, 11347, 11601, 11602, 11603) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Godwars Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
